models/user.js

```diff
@@ -9,6 +9,7 @@ module.exports = function(sequelize, DataTypes) {
         password: DataTypes.STRING,
```

Comment:
We won't need to store the user's password because the Auth is handled by the Facebook SDK. We should store the user's Facebook userID instead.

Reply:
True. But I still added this and the other redundant field "notes" as placeholders. This gives us the flexibility to ignore these fields for now but use them in the future if we have to.